pkg/cmd/release/deploy/deploy.go

```diff
@@ -227,9 +221,9 @@ func deployRun(cmd *cobra.Command, f factory.Factory, flags *DeployFlags) error
 	}
 
 	if f.IsPromptEnabled() {
-		now := time.Now()
+		now := time.Now
```

Comment:
Previously it captured 'now' at the start of the process, so if you took ages answering questions, it would be wrong by the time you got to the scheduled start date. Fixed by capturing a function pointer which we don't call until the last possible time